src/Cli.elm

```diff
@@ -75,8 +75,9 @@ program =
                         |> Cli.Option.validateMap serverValidation
                     )
                 |> Cli.OptionsParser.withDoc """
+version: 0.5.0
```

Comment:
Wild idea: add an elm-review rule to check that this stays in sync with `elm.json`

Reply:
We could do that now!

Reply:
I see you've split this off in #117